src/lib/rule-finder.js

```diff
@@ -103,6 +108,6 @@ function RuleFinder(specifiedFile, noCore) {
   this.getUnusedRules = () => getSortedRules(unusedRules);
 }
 
-module.exports = function (specifiedFile, noCore) {
-  return new RuleFinder(specifiedFile, noCore);
+module.exports = function (specifiedFile, options = {}) {
```

Comment:
CI is complaining, because the branch where `RuleFinder` assigns the default value to its `options` parameter never gets called (because the exported function already assigns an empty object as default value to `option`).

It is understandable, that this fails ... this should not have been passing previously.